LiskHQ/lisk-desktop
# Fix staking amount validation error

### What was the problem?

This PR resolves #5122 

### How was it solved?

- [x] Fix amount validation in the `useStakeAmountField` hook

### How was it tested?

- Navigate to the validator's page
- Try to add or edit stakes on any validator
- Staking amount which should be the difference between the confirm and unconfired amount should now be properly validated against total wallet balance.
